tests/cypress/support/commands.js

```diff
@@ -1431,6 +1431,82 @@ Cypress.Commands.add('finishMaskDrawing', () => {
     cy.get('.cvat-brush-tools-finish').click();
 });
 
+Cypress.Commands.add('sliceTool', (
```

Comment:
I would suggest `sliceShape`
`sliceTool` sounds like a noun, command usually is action (verb)